client/src/apis/index.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
```

Comment:
Error: `Module not found: Can't resolve 'axios'`

Please install the package first and add instructions in PR description.